GeometrickFX/Shapes/AbstractDomain/ShapeGlobals.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shapes.AbstractDomain
+{
+    public abstract class ShapeGlobals
```

Comment:
Why shouldn't you combine IShape with ShapeGlobals?

Reply:
removed shape globals